### DISCUSSION TITLE
dht-only seeding seems not work between two rtorrent clients.

### DISCUSSION THREAD

**Discussion by sezuan** [rakshasa/rtorrent#1093] [Q&A] [answered]
Hello,

I'm playing around with dht. Here's the setup and what's working:

1. Two rtorrent clients, both with public ipv4/ipv6. Required ports are open and verified with netcat.
2. Both rtorrents can leech with magent urls where the tracker url is removed. When I check the tracker list the transfer seems to continue without tracker.
3. The Peer list indicates that there are incoming and outgoing connections.

What's not working:

1. Create a new and unique trackerless torrent.
2. One rtorrent client is seed, the other leech.
3. -> the transfer never starts. no dht errors. debug log doesn't give me a clue.
4a. add a public tracker to both torrents -> transfer starts.
4b. OR try to download the trackerless torrent with utorrent(android) -> transfer starts.

Any ideas how to debug this? I'm using rtorrent rTorrent 0.9.7/0.13.7 (debian buster version)

**Comment by rakshasa** [comment, marked as answer]
This is not supported atm.

**Reply by sezuan** [reply]
Can you elaborate a bit and point me in the right direction. I'm just curious what functionality is missing. Can't rtorrent deliver the torrent file to a client that just has the magnet url?

**Reply by rakshasa** [reply]
Just a magnet URL should work.

It's been a decade since I really reviewed the DHT code, so probably misremembering stuff, however I think there was some issues with just rtorrent initializing a trackerless torrent.
